src/test/kotlin/com/zepben/evolve/database/sqlite/cim/upgrade/changesets/network/ChangeSet55NetworkValidator.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.cim.upgrade.changesets.network
+
+import com.zepben.evolve.database.paths.DatabaseType
+import com.zepben.evolve.database.sqlite.cim.upgrade.changesets.ChangeSetValidator
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import java.sql.Statement
+
+object ChangeSet55NetworkValidator : ChangeSetValidator(DatabaseType.NETWORK_MODEL, 55) {
+
+    //
+    // NOTE: In the validators we are only checking the columns that were actually changed.
+    //
+
+    override fun setUpStatements(): List<String> = emptyList()
```

Comment:
todo: Missing setup statements. I have updated the docstrings to add clarity to the expectation for this (and the other) calls.

If you do this correctly, you should break your `UsagePoint` test below

Reply:
Changed up validator and also added a default to alter table.